### DISCUSSION TITLE
is there a pnpm run equivalent for `run-s` / `npm-run-all -s`?

### DISCUSSION THREAD

**Discussion by stefanprobst** [pnpm/pnpm#6099] [Q&A]
with the support for regexes added to `pnpm run` in v7.27 it should be possible to replace `run-p`/`npm-run-all -p` -- i'm curious if there is an equivalent for `run-s`/`npm-run-all -s` as well? currently i have scripts like `run-s format lint typecheck`.

**Comment by Jintus** [comment]
Did you have an answer ?

At the moment for your exemple i do : `pnpm format && pnpm lint && pnpm typecheck`
